app/src/main/java/com/android/shelfLife/model/recipe/Recipe.kt

```diff
@@ -13,18 +13,25 @@ data class Recipe(
     val servings: Float, // total number of servings
     val time: Duration, // time it takes to cook
     val ingredients: List<Ingredient> = listOf(), // ingredients in recipe
-    val recipeType: RecipeType = RecipeType.USE_SOON_TO_EXPIRE
-)
+    val recipeType: RecipeType = RecipeType.PERSONAL,
+    val workInProgress: Boolean = false, // if the recipe is currently being worked on
+) {
+  companion object {
+    const val MAX_SERVINGS: Float = 20.0f
+  }
+}
 
 /** Data class representing a recipe prompt, that we use to query the Recipe generation model */
 data class RecipePrompt(
     val name: String,
-    val recipeType: RecipeType = RecipeType.USE_SOON_TO_EXPIRE,
+    val recipeType: RecipeType = RecipeType.BASIC,
     val specialInstruction: String = "",
     val ingredients: List<FoodItem> = listOf(),
     val missingIngredients: List<String> = listOf(),
     val servings: Float = 1.0f,
     val shortDuration: Boolean = false,
+    val onlyHouseHoldItems: Boolean = false,
+    val prioritiseSoonToExpire: Boolean = true,
```

Comment:
I much prefer the boolean way for using soon to expire items, it is much cleaner and simpler than using a recipe type. Good job!!